plugins/modules/system/xfconf_info.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2021, Alexei Znamensky <russoz@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+module: xfconf_info
+author:
+    - "Alexei Znamensky (@russoz)"
+short_description: Retrieve XFCE4 configurations
+description:
+  - This module allows retrieving Xfce 4 configurations via xfconf-query.
+options:
+  channel:
+    description:
+    - >
+      A Xfconf preference channel is a top-level tree key, inside of the
+      Xfconf repository that corresponds to the location for which all
+      application properties/keys are stored.
+    - If not provided, the module will list all available channels.
+    type: str
+  property:
+    description:
+    - >
+      A Xfce preference key is an element in the Xfconf repository
+      that corresponds to an application preference.
+    - If provided, then I(channel) is required.
+    - If not provided and a I(channel) is provided, then the module will list all available properties in that I(channel).
+    type: str
+notes:
+  - See man xfconf-query(1) for more details.
+'''
+
+EXAMPLES = """
+- name: Get list of all available channels
+  community.general.xfconf_info: {}
```

Comment:
For info modules I think it's always nice to show that you should register the results.
```suggestion
  community.general.xfconf_info: {}
  register: result
```